build.sbt

```diff
@@ -127,9 +128,17 @@ val identity = application("identity").dependsOn(commonWithTests).aggregate(comm
   )
 )
 
-val commercial = application("commercial").dependsOn(commonWithTests).aggregate(common)
+val commercial = application("commercial").dependsOn(commonWithTests).aggregate(common).settings(
+  libraryDependencies ++= Seq(
+    dispatch
+  )
+)
 
-val onward = application("onward").dependsOn(commonWithTests).aggregate(common)
+val onward = application("onward").dependsOn(commonWithTests).aggregate(common).settings(
+  libraryDependencies ++= Seq(
+    dispatch
+  )
+)
```

Comment:
I don't understand why adding dispatch is related to this snippet PR? Could you give some details? Thank you

Reply:
The latest version of `scala-client` no longer supports Dispatch. Onward, Commercial & Identity previously resolved their dependancy via the client, so to get the build to pass it needs to be added back in.

I saw that @mario-galic removed dispatch in Identity so perhaps something similar could be done for Onward & Commercial.

I appreciate this PR is resolving a lot of things besides a description field on a snippet! However the `scala-client` version on frontend was several versions behind so needed to do some additional lifting to get in a workable state. LMK if you have any other questions.

Reply:
Ok I understand. So `dispatch` was used by `onward` and `commercial` but not specified explicitely in those apps settings. That's bad.
Sorry for the trouble.

Reply:
no worries! i think it was a question of when each dependancy was loaded for Play deciding which copy to use, so even in Identity with the dispatch dependancy explicit it was using the scala-client version.